src/main/java/com/epam/lowcost/controller/FlightController.java

```diff
@@ -60,21 +74,35 @@ public String addNewFlight() {
         return "addFlight";
     }
 
-    @GetMapping(value = FLIGHT)
-    public String searchForFlight(Model model) {
-        model.addAttribute("flights", ((FlightServiceImpl) flightService).getAllFlightsWithUpdatedPrice());
+    @GetMapping(value = FLIGHT+ "/{pageId}")
+    public String searchForFlight(@PathVariable int pageId, ModelMap model) {
+        //model.addAttribute("flights", ((FlightServiceImpl) flightService).getAllFlightsWithUpdatedPrice());
+
+        int flightsByPage = (int) model.getOrDefault("number", 2);
```

Comment:
what does it mean, 2 ?

Reply:
Number of flights to show on one page by default.

Reply:
move into separate variable, declaration of similar variables should be obvious

Reply:
Changed to more appropriative variable names.